_posts/en/newsletters/2024-12-13-newsletter.md

```diff
@@ -0,0 +1,308 @@
+  [suggested][towns poll] simply asking developers whether they had any
+  outstanding concerns about each proposal.  Both Nick and Towns
+  recommended developers link to evidence and arguments supporting their
+  opinions.
+
+  Although faults in the poll were highlighted, a showing of greater
+  support for some proposals over other proposals may help covenant
+  researchers converge on a short list of ideas for the broader
+  community to review.
+
+- **Incentive-based pseudo-covenants:** Jeremy Rubin [posted][rubin
+  unfed] to the Bitcoin-Dev mailing list a link to a [paper][rubin unfed
+  paper] he authored about oracle-assisted [covenants][topic covenants].
+  The model involves two oracles: a _covenant oracle_ and an _integrity
+  oracle_.  The covenant oracle places funds in a fidelity bond and
+  agrees to only sign transactions based on a program returning success.
+  The integrity oracle can seize funds from a bond by using [accountable
+  computing][topic acc] to prove a signature was created for a program
+  that did not return success.  If fraud occurs, there is no guarantee
+  that a user who lost money due to the covenant oracle's deceit will
+  recover lost funds.
+
+  Ethan Heilman independently [posted][heilman slash] to the Bitcoin-Dev
+  mailing list a different construction that allows anyone to use a
+  fraud proof to punish incorrect signing.  However, in this case, the
+  funds are not seized but destroyed.  This ensures a fraudulent signer
+  is punished but prevents any victims from recovering their lost value.
+
+- **Bitcoin Core developer meeting summaries:** FIXME:harding, see https://btctranscripts.com/bitcoin-core-dev-tech/2024-10
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+FIXME:stickies-v
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/30239#l-27FIXME"
+%}
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Java-based HWI released:**
+  [Lark App][larkapp github] is a command line application for interacting with
+  hardware signing devices. It uses the [Lark Java library][lark github], a port
+  of [HWI][topic hwi] for the Java programming language.
+
+- **Saving Satoshi Bitcoin development education game announced:**
+  The [Saving Satoshi][saving satoshi website] website provides interactive
+  educational exercises for those new to Bitcoin development.
+
+- **Neovim Bitcoin Script plugin:**
+  The [Bitcoin script hints][bsh github] Neovim plugin for Rust shows Bitcoin
+  script stack states for each operation within the editor.
+
+- **Proton Wallet adds RBF:**
+  Proton Wallet users can now [RBF][topic rbf] fee bump [their
```

Comment:
```suggestion
  Proton Wallet users can now [RBF][topic rbf] fee-bump [their
```

Reply:
We have fee bumping vs fee-bumping in the style guide, but perhaps the hyphen is appropriate here, defer to @harding

Reply:
Or another idea (I should have suggested this earlier):
```suggestion
  Proton Wallet users can now use [RBF][topic rbf] to fee-bump [their
```